crates/eip7594/src/trusted_setup.rs

```diff
@@ -132,31 +132,26 @@ fn deserialize_g1_points<T: AsRef<str>>(
     g1_points_hex_str: &[T],
     check: SubgroupCheck,
 ) -> Vec<G1Point> {
-    let mut g1_points = Vec::new();
-    for g1_hex_str in g1_points_hex_str {
-        let g1_hex_str = g1_hex_str.as_ref();
-
-        let g1_hex_str_without_0x = g1_hex_str
-            .strip_prefix("0x")
-            .expect("expected hex points to be prefixed with `0x`");
-        let g1_point_bytes: [u8; 48] = hex::decode(g1_hex_str_without_0x)
-            .expect("trusted setup has malformed g1 points")
-            .try_into()
-            .expect("expected 48 bytes for G1 point");
-
-        let point = match check {
-            SubgroupCheck::Check => {
-                G1Point::from_compressed(&g1_point_bytes).expect("invalid g1 point")
+    g1_points_hex_str
+        .iter()
+        .map(|s| {
+            let s = s
+                .as_ref()
+                .strip_prefix("0x")
+                .expect("expected hex points to be prefixed with `0x`");
+
+            let bytes: [u8; 48] = hex::decode(s)
```

Comment:
```suggestion
        .map(|hex_str| {
            let hex_str = hex_str
                .as_ref()
                .strip_prefix("0x")
                .expect("expected hex points to be prefixed with `0x`");

            let bytes: [u8; 48] = hex::decode(hex_str)
```

Small nit: to avoid one letter variables when possible!